Traceback (most recent call last):
  File "/Users/jarv/.virtualenvs/ratetheinterview/bin/letsencrypt", line 11, in <module>
    sys.exit(main())
  File "/Users/jarv/.virtualenvs/ratetheinterview/lib/python2.7/site-packages/certbot/main.py", line 740, in main
    os.geteuid(), config.strict_permissions)
  File "/Users/jarv/.virtualenvs/ratetheinterview/lib/python2.7/site-packages/certbot/main.py", line 726, in make_or_verify_core_dir
    raise errors.Error(_PERM_ERR_FMT.format(error))
Error: The following error was encountered:
[Errno 13] Permission denied: '/etc/letsencrypt'
If running as non-root, set --config-dir, --logs-dir, and --work-dir to writeable paths.
